packages/react/src/components/organization-switcher/index.tsx

```diff
@@ -10,6 +10,7 @@ import { OrganizationDetails } from "@slashid/slashid";
 interface Props {
   fallback?: ReactNode;
   filter?: (organization: OrganizationDetails) => boolean;
+  renderLabel?: (organization: OrganizationDetails) => ReactNode;
```

Comment:
I intentionally did not name this `renderItem` because it does not allow you to control the full rendering of the menu item only the label.

The selected item for example has a checkmark, we don't allow you to configure this (yet).

I anticipate in future we would also have `renderItem` which would allow control of the entire item renderer, but out of scope for now.